TeamCode/src/main/java/org/firstinspires/ftc/teamcode/MecanumDrive.java

```diff
@@ -199,16 +205,16 @@ public Twist2dDual<Time> update() {
 
             lastHeading = heading;
 
-            return new Twist2dDual<>(
+            pose = pose.plus(new Twist2dDual<>(
```

Comment:
I think you can just do `pose = pose.plus(new Twist2d(twist.line.value(), headingDelta))`

Reply:
Does it matter? (genuine question)

Reply:
Yes? It's like having `Vector2d(x, y).x + 2` instead of `x + 2` to me.